portfolio/src/main/webapp/script.js

```diff
@@ -15,17 +15,19 @@
 /**
  * Adds a random activity I might be doing to the page
  */
+
+const activities = ['Coding 💻', 'Travelling 🗼', 'Brewing coffee ☕', 'Going on a nature walk 🌲'];
```

Comment:
Make sure the comment stays with the function! When you add a global variable, give it its own comment.

Reply:
Thanks - Fixed!